app/tests/unit/pages/project-revision/[projectRevision]/index.test.tsx

```diff
@@ -267,4 +271,34 @@ describe("The Create Project page", () => {
       },
     });
   });
+
+  it("renders a disabled submit / discard button when project revision mutations are in flight", async () => {
+    const mockProjectForm: any = { props: {} };
+    jest
+      .spyOn(require("components/Form/ProjectForm"), "default")
+      .mockImplementation((props) => {
+        mockProjectForm.props = props;
+        return null;
+      });
+
+    jest
+      .spyOn(require("react-relay"), "useMutation")
+      .mockImplementation(() => [jest.fn(), true]);
+
+    render(
+      <RelayEnvironmentProvider environment={environment}>
+        <ProjectRevision
+          data-testid="2"
+          CSN={true}
```

Comment:
FYI I figured out what that CSN prop is. It's supposed to be whether or now we're on the client, and will determine whether a `Suspense` is rendered or not